test/codeu/chat/server/JSONTest.java

```diff
@@ -0,0 +1,204 @@
+package codeu.chat.server;
+import static org.junit.Assert.*;
+
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+import org.junit.Ignore;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.io.IOException;
+import codeu.chat.util.store.Store;
+import java.lang.Iterable;
+import java.util.Iterator;
+import codeu.chat.util.Time;
+
+/**
+ * This class tests the JSON class in the server folder
+ * The purpose of this test is to ensure that the user is able to
+ * access the transaction log and that the transaction log is
+ * being saved properly.
+ */
+public class JSONTest {
+
+    @Test
+    public void readTest1() {
```

Comment:
I would rename your test functions to be more obvious. Something like `testReadAll` or `testReadUser`, names like that.